fe/fe-core/src/main/java/org/apache/doris/common/util/TimeUtils.java

```diff
@@ -149,13 +152,22 @@ public static String longToTimeString(long timeStamp, SimpleDateFormat dateForma
         return dateFormat.format(new Date(timeStamp));
     }
 
-    public static String longToTimeString(long timeStamp) {
-        SimpleDateFormat datetimeFormatTimeZone = datetimeFormatThreadLocal.get();
+    public static String longToTimeStringProc(long timeStamp, SimpleDateFormat datetimeFormatTimeZone) {
```

Comment:
longToTimeStringProc, the name is a little strange. Maybe we can rename it to longToTimeStringWithFormat?